test/backend_fused_op.in.cpp

```diff
@@ -420,3 +421,52 @@ NGRAPH_TEST(${BACKEND_NAME}, gemm_broadcast_input_C)
     test_case.add_expected_output<double>(Shape{3, 4}, vector<double>(12, 7));
     test_case.run();
 }
+
+NGRAPH_TEST(${BACKEND_NAME}, squeeze)
+{
+    auto data_node = make_shared<op::Parameter>(element::f32, Shape{1, 4, 1, 1, 2});
```

Comment:
In case you add more commits, please `const` everything you can in all of those tests